src/main/java/frc/robot/commands/elevator/ElevatorToLimitSwitchCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.elevator.ElevatorSubsystem;
+
+/** Sends the elevator to position zero. */
+public class ElevatorToLimitSwitchCommand extends Command {
```

Comment:
Can you also combine this to be a part of the toGround command? Maybe you can just make a command that is a sequence of first doing the toGround and then this command and we will use that to lower the elevator.

Reply:
So first do toGround command and then if it doesn't touch the limit switch I keep lowering it?